repositories.yaml

```diff
@@ -238,3 +238,11 @@ mozregression:
     - 'mozregression/metrics.yaml'
   ping_files:
     - 'mozregression/pings.yaml'
+firefox-ios:
+  app_id: org-mozilla-ios-fennec
+  notification_emails:
+    tlong@mozilla.com
+    frank@mozilla.com
+  url: https://github.com/mozilla-mobile/firefox-ios
+  metrics_files:
+    - 'Client/Generated/LegacyIds.swift'
```

Comment:
I think this should point to the metrics.yaml, not the generated code.

Reply:
Indeed it definitely should... strange that I missed that 🤦‍♀️